### PR TITLE
Add form in the UI

### PR DESCRIPTION
**Pull request by Aqurds**
The following features were added,

**Create a method called current_user in your ApplicationController to make current user data available to all controllers.**
- > It will return the first user from the database.

**Create forms to perform the following functions:**
- > Creates a Post on behalf of the current_user.
- > Create a comment on behalf of the current_user.
- > Allow Users to add likes to Posts.

### CODE REVIEW

File: app/controllers/application_controller.rb
Comment (Felix45) [inline app/controllers/application_controller.rb:1]: - [ Required ] Good job working on this project, you did well :+1: . However, I've noticed that the `LICENSE` link on your readme file is broken and it leads to a `404` page not found error. A working License link ensures that anyone who would like to use your code is well-informed and knows on how to use it within the law. I would like to suggest that you refactor the License link as shown below

## Example code
> The word `LICENSE` is currently not spelled correctly on your readme file. Kindly rectify this.
```ruby
This project is [MIT](./LICENSE) licensed.
```

**Review by Felix45 — CHANGES_REQUESTED** [review]
Hi @Aqurds, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights :heavy_check_mark: 
- [x] Good UI design :white_check_mark: 
- [x] All application forms work as expected :white_check_mark: 
- [x] Pull request has a descriptive title and summary :white_check_mark: 
- [x] Git flow is used as expected :white_check_mark:    


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/application_controller.rb
Comment (Aqurds) [reply app/controllers/application_controller.rb:1]: Readme updated. Thanks for pointing out :)

**Review by divinecharlotte — APPROVED** [review]
# Hello @Aqurds   😃 🥳 🎉 🎈 

Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 
### To Highlight 💯

- Good implementation of the required changes✔️
- Descriptive pr✔️






## Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**